cmd/brew-createzap

```diff
@@ -95,7 +95,7 @@ generate() {
     echo "zap trash: \"${path/$HOME/~}\""
   else
     echo "zap trash: ["
-    cat $TMP_FILE | while read path
+    cat $TMP_FILE | sort | while read path
```

Comment:
Results should always be sorted alphabetically. Right now, they are most of the time due to `TRASH_PATHS` being in alphabetical order, but not always.

Before:
```
% brew createzap jupyter
zap trash: [
  "~/Library/Jupyter",
  "~/Library/Application Support/jupyterlab-desktop",
  "~/Library/Logs/JupyterLab",
  "~/Library/Logs/jupyterlab-desktop",
  "~/Library/Preferences/com.electron.jupyterlab-desktop.plist",
  "~/Library/Saved Application State/com.electron.jupyterlab-desktop.savedState",
  "~/.jupyter",
]
```
After:
```
% brew createzap jupyter
zap trash: [
  "~/.jupyter",
  "~/Library/Application Support/jupyterlab-desktop",
  "~/Library/Jupyter",
  "~/Library/Logs/JupyterLab",
  "~/Library/Logs/jupyterlab-desktop",
  "~/Library/Preferences/com.electron.jupyterlab-desktop.plist",
  "~/Library/Saved Application State/com.electron.jupyterlab-desktop.savedState",
]
```